<launch>
  <arg name="model" />
  <arg name="gui" default="true"/>
  <param name="robot_description" textfile="$(find gimbal_urdf)/urdf/gimbal_urdf.urdf" />
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
    <rosparam param="source_list">["ik_joint_states"]</rosparam>
    <param name="use_gui" value="$(arg gui)"/>
  </node>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find gimbal_urdf)/urdf.rviz" />

    <!-- reference object -->
  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="1 1 -1 0 0 1 0 base_link sagebrush 100" />


</launch>